Source/WebCore/page/UserStyleSheetTypes.h

```diff
@@ -25,14 +25,16 @@
  
 #pragma once
 
-#include <wtf/EnumTraits.h>
 #include <wtf/HashMap.h>
 #include <wtf/Vector.h>
 
 namespace WebCore {
 
 enum UserStyleInjectionTime { InjectInExistingDocuments, InjectInSubsequentDocuments };
-enum UserStyleLevel { UserStyleUserLevel, UserStyleAuthorLevel };
+enum class UserStyleLevel : bool {
```

Comment:
nit: could have stayed on a single line, it is our preferred style for small enums.

Reply:
OK.